package.json

```diff
@@ -11,6 +11,7 @@
   },
   "devDependencies": {
     "@newrelic/eslint-plugin-newrelic": "^0.3.1",
+    "@testing-library/jest-dom": "^5.11.4",
```

Comment:
would you be able to explain what `jest-dom` is doing here?

Reply:
[`jest-dom`](https://github.com/testing-library/jest-dom) adds a bunch of really useful matchers to match against DOM-specific stuff. For example, [`.toBeVisible()`](https://github.com/testing-library/jest-dom#tobevisible) or [`.toBeInTheDocument()`](https://github.com/testing-library/jest-dom#tobeinthedocument) (which I'm using [in this PR](https://github.com/newrelic/gatsby-theme-newrelic/pull/100/files#diff-c6211204be539afd5bf8ed94af3a1f1dR46).)

Check out the documentation for the full list of matchers!